tests/test_modeling_tf_common.py

```diff
@@ -129,7 +129,7 @@ def test_save_load(self):
 
                 self.assert_outputs_same(after_outputs, outputs)
 
-    @slow
+    @unittest.skip("Skipping this as too slow")
```

Comment:
Will reinstate those later, but they're way too slow for now.

Reply:
making a new - `@tooslow` decorator?